### PR TITLE
Adding Traffic Violation example  - Replace Edson's PR

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

**Comment by porcelli** [comment]
This PR depends on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/927

**Comment by porcelli** [comment]
jenkins execute full downstream build

File: traffic-violation/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.workbench.playground</groupId>
+    <artifactId>playground-parent</artifactId>
+    <version>7.18.0-SNAPSHOT</version>
```
Comment (jomarko) [inline traffic-violation/pom.xml]: 7.19.0-SNAPSHOT

**Comment by jomarko** [comment]
screenshot from 2019-02-19 11-24-43

I found just one thing to discuss, if we have a look on list on available samples, this new one follows different naming. Existing samples are uppercase with _ instead of - between words. If you think it is not an issue I am fine with merging.

**Comment by porcelli** [comment]
changes applied @jomarko

**Comment by jomarko** [comment]
@porcelli thank you

**Comment by porcelli** [comment]
updated as requested by @danielezonca

**Comment by jomarko** [comment]
I see changes landed in since my review, however due to missing resources I am not going to re-review.

**Comment by danielezonca** [comment]
@porcelli 
Can you please close this? I created a new PR (https://github.com/kiegroup/kie-wb-playground/pull/75) so I can easier fix errors

**Pull request closed without merging**